akka-http-core/src/main/scala/akka/http/impl/engine/rendering/HttpResponseRendererFactory.scala

```diff
@@ -33,22 +33,22 @@ private[http] class HttpResponseRendererFactory(serverHeader: Option[headers.Ser
     }
 
   // as an optimization we cache the Date header of the last second here
-  @volatile private[this] var cachedDateHeader: (Long, Array[Byte]) = (0L, null)
-
+  private[this] var cachedDateHeader: (Long, Array[Byte]) = (0L, null)
```

Comment:
I guess we could indeed get rid of the `volatile` here and simply allow for some fuzzyness.
Updates lost to the missing publishing would simply cause a bit more work on another core, which would hopefully be outweighed by removing the `volatile` overhead.
We might even split the `Long` and the bytes and remove the tuple.